gobblin-kubernetes/gobblin-service/dev-cluster/mysql-deployment.yaml

```diff
@@ -0,0 +1,55 @@
+apiVersion: v1
```

Comment:
How do we organize the content between `fs` based and `mysql` based? Using the same parent folder `dev-cluster` is confusing. 

Currently, we're using `kustomization` file to share and organize k8s application configs. I guess we'll have a mysql folder with its own `kustomization` file?